app/controllers/responses_controller.rb

```
@@ -0,0 +1,115 @@
+    unless @map
+      render json: { error: "ResponseMap not found" }, status: :not_found
+      return
+    end
+
+    unless @map.reviewer.user_id == current_user.id
+      render json: { error: "You are not authorized to create this responses" }, status: :forbidden
+    end
+  end
+ 
+
+  def response_update_params
```

RuboCop (1.86.1)

[convention] Expected 1 empty line between method definitions; found 2.

(Layout/EmptyLineBetweenDefs)